src/main/java/me/jellysquid/mods/sodium/client/render/chunk/ChunkRenderManager.java

```diff
@@ -82,8 +78,23 @@
 
     private int visibleChunkCount;
 
+    public static class RenderContext<T1 extends ChunkGraphicsState> {
+        public final ObjectArrayFIFOQueue<ChunkRenderContainer<T1>> iterationQueue = new ObjectArrayFIFOQueue<>();
+        @SuppressWarnings("unchecked")
+        public final ChunkRenderList<T1>[] chunkRenderLists;
+        public final ObjectList<ChunkRenderContainer<T1>> tickableChunks = new ObjectArrayList<>();
+        public final ObjectList<BlockEntity> visibleBlockEntities = new ObjectArrayList<>();
+
+        public RenderContext(int renderPassCount) {
+            chunkRenderLists = new ChunkRenderList[renderPassCount];
+            for (int i = 0; i < this.chunkRenderLists.length; i++) {
+                this.chunkRenderLists[i] = new ChunkRenderList<>();
+            }
+        }
+    }
+
     @SuppressWarnings("unchecked")
-    public ChunkRenderManager(SodiumWorldRenderer renderer, ChunkRenderBackend<T> backend, ClientWorld world, int renderDistance) {
+    public ChunkRenderManager(SodiumWorldRenderer renderer, ChunkRenderBackend<T> backend, BlockRenderPassManager renderPassManager, ClientWorld world, int renderDistance) {
```

Comment:
Why was `BlockRenderPassManager renderPassManager` added here? No code seems to use it, and it can be obtained through a call to `ChunkRenderBackend.getRenderPassManager()`